src/runtime_src/core/edge/include/sk_types.h

```diff
@@ -45,29 +47,11 @@ struct sk_operations {
  *       for soft kernel to run.
  * ops:  provide help functions for soft kernel to use.
  */
+XRT_DEPRECATED
 typedef int (* kernel_t)(void *args, struct sk_operations *ops);
 
-/*
- * PS Context Data Structure included by user PS kernel code
- */
-
-class pscontext {
-public:
- pscontext()
-   : pimpl{std::make_shared<pscontext::impl>()} {}
-  virtual ~pscontext() {}
- 
-protected:
-  struct impl;
-  std::shared_ptr<impl> pimpl;
-};
-
-struct pscontext::impl {
-private:
-  bool aie_profile_en;
-};
+#pragma message ("sk_types.h is deprecated and will be removed from the distribution directory in a future release.  Please use pscontext.h instead.")
 
-typedef pscontext* (* kernel_init_t)(xclDeviceHandle device, const uuid_t &uuid);
-typedef int (* kernel_fini_t)(pscontext *xrtHandles);
+#include "pscontext.h"
```

Comment:
Can add a comment here saying this include is for backward compatibility?

Reply:
Comment added